opengrok-web/src/main/webapp/repos.jspf

```diff
@@ -156,13 +156,28 @@ Copyright (c) 2015, 2018, Oracle and/or its affiliates. All rights reserved.
                             if (cnt > 0 && ri.getParent() == null)
                                 // discard repositories without a parent url
                                 continue;
-                            if (cnt != 0) {
+
+                            boolean subrepository = !ri.getDirectoryNameRelative().equals(project.getPath());
+                            if(subrepository && cnt == 0) {
```

Comment:
missing space between `if` and left paren ?

Reply:
yes